src/web/routes/application/address/postcode/postcode.test.js

```diff
@@ -126,6 +129,19 @@ test('behaviourForPost() handles address lookup error', async (t) => {
   t.end()
 })
 
+test('behaviourForPost() does not call os places when there are validation errors', (t) => {
+  const req = {}
+  const res = {}
+  const next = () => {}
+  request.reset()
+  validationResult.onCall().returns(['error'])
```

Comment:
Do you need `onCall` here? I thought it could just be:
```
validationResult.returns(['error'])
```
... but I might be wrong.

Reply:
You're right, it's not needed.